docs/view_components/menu_bar.rst

```diff
@@ -1,11 +1,59 @@
-Component title
----------------
+.. _menu_bar-label:
+
+Menu Bar
+--------
+
+Menu bar is a tab with a set of menus namely File, Scenarios, Preferences and Help.
 
 Attributes
 ^^^^^^^^^^
 
+* The following are the menus available in the menu bar:
+    #. File
+    #. Scenarios
+    #. Preferences
+    #. Help
+* *File* has the following submenus or options:
+    I. Load project
+    II. New project
+    III. Save project
+    IV. Save project as
+    V. Export project
+    VI. Exit open_plan
+* *Scenarios* has the following submenus:
+    I. Load scenario
+    II. New scenario
+    III. Current scenario
+* *Preferences* has the following submenus:
+    I. Language
```

Comment:
I think we are only going to provide english, or should we make sure we have the option to have it also in german @catcad ?

Reply:
If it is not a difficult task (i.e., if there is some library or some sort of tool that lets us do this without much coding), I support providing the tool in German and other possible languages as well